src/components/Form/FormWrapper.tsx

```diff
@@ -121,26 +121,26 @@ function FormWrapper({
             </FormElement>
         ),
         [
-            children,
-            enabledWhenOffline,
-            errorMessage,
-            errors,
-            footerContent,
             formID,
-            formState?.errorFields,
-            formState?.isLoading,
-            isSubmitActionDangerous,
-            isSubmitButtonVisible,
-            onSubmit,
             style,
             styles.flex1,
             styles.mh0,
             styles.mt5,
+            children,
```

Comment:
Why are we changing the order of these props?

Reply:
Bump on this comment

Reply:
@eVoloshchak I don't recall why I changed this 😅 I'm going to revert this changes, thank you for noticing it!